packages/mgt-components/src/components/mgt-file-list/mgt-file-list.ts

```diff
@@ -41,10 +41,13 @@ import { strings } from './strings';
 import { MgtFile } from '../mgt-file/mgt-file';
 import { MgtFileUploadConfig } from './mgt-file-upload/mgt-file-upload';
 
-import { fluentProgressRing } from '@fluentui/web-components';
-import { registerFluentComponents } from '../../utils/FluentComponents';
+export { FluentDesignSystemProvider, FluentProgressRing } from '@fluentui/web-components';
+export * from './mgt-file-upload/mgt-file-upload';
```

Comment:
Is this needed?

Reply:
Yes, this helps to register the fluent components used by file upload